main.py

```diff
@@ -1,7 +1,47 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors_fullname = [("George", "Klooney"), ("Kianu", "Reaves"),
+                       ("Scarlett", "Keegan"), ("Will", "Smith"),
+                       ("Jaden", "Smith"), ("Scarlett", "Johansson")]
```

Comment:
The list `actors_fullname` contains tuples with actor names. Ensure that the names are spelled correctly, as there are typos in 'Klooney' and 'Kianu'. These are corrected later in the code, but it's better to have them correct from the start.